lottery.py

```diff
@@ -1,24 +1,23 @@
 from random import choice
 
-# Did some changes
 class Lottery:
 
-    def __init__(self, lottery_codes):
+    def __init__(self, lottery_codes, min_code_length, max_code_length):
```

Comment:
наступне 'покращення' - дефолтні знаяення для `min_code_length`  та `max_code_length`  брати з констант
```suggestion
    def __init__(self, lottery_codes, min_code_length=MIN_CODE_LENGTH, max_code_length=MAX_CODE_LENGTH):
```